### PR TITLE
Add k3d platform for ambient

### PR DESCRIPTION
**Pull request by linsun**
## Description

Finally got k3d working on my mac with @bleggett's help, document the caveat.

## Reviewers

- [ x ] Ambient
- [x ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

### CODE REVIEW

File: content/en/docs/ambient/install/platform-prerequisites/index.md [hunk 1]
```diff
@@ -47,9 +47,24 @@ and capture pods on the node.
 
 1. If you are using [MicroK8s](https://microk8s.io/), you must append `--set values.cni.cniConfDir=/var/snap/microk8s/current/args/cni-network --set values.cni.cniBinDir=/var/snap/microk8s/current/opt/cni/bin` to the `helm install` command, as MicroK8s [uses nonstandard locations for CNI configuration and binaries](https://microk8s.io/docs/change-cidr).
 
+### K3D
+
+1. If you are using [k3d](https://k3d.io/) - a lightweight wrapper to run [k3s](https://github.com/rancher/k3s), you must append `--set values.cni.cniConfDir=/var/lib/rancher/k3s/agent/etc/cni/net.d --set values.cni.cniBinDir=/bin/` to your `istioctl install` or `helm install` command to install Istio with the `ambient` profile.
```
Comment (bleggett) [inline content/en/docs/ambient/install/platform-prerequisites/index.md:52]: Believe that the replaced text is still correct, and that this is only true if you are using the default CNI that `k3d` ships with (flannel).

If you were using calico and disabled the default CNI, then the standard paths would be required.

So this should probably still read "if you are using k3s with the default flannel CNI"

Comment (linsun) [reply content/en/docs/ambient/install/platform-prerequisites/index.md]: Thanks, clarified this is for default flannel CNI

**Comment by linsun** [comment]
Hi @ericvn Could you review this? Thanks!

**Review by ericvn — APPROVED** [review]

File: content/en/docs/ambient/install/platform-prerequisites/index.md [hunk 2]
```diff
@@ -47,9 +47,24 @@ and capture pods on the node.
 
 1. If you are using [MicroK8s](https://microk8s.io/), you must append `--set values.cni.cniConfDir=/var/snap/microk8s/current/args/cni-network --set values.cni.cniBinDir=/var/snap/microk8s/current/opt/cni/bin` to the `helm install` command, as MicroK8s [uses nonstandard locations for CNI configuration and binaries](https://microk8s.io/docs/change-cidr).
 
+### K3D
+
+1. If you are using [k3d](https://k3d.io/) with the default flannel CNI, you must append `--set values.cni.cniConfDir=/var/lib/rancher/k3s/agent/etc/cni/net.d --set values.cni.cniBinDir=/bin/` to your `istioctl install` or `helm install` command to install Istio with the `ambient` profile.
+
+1. Create a cluster and disable `Traefik` with the following command:
```
[the remark this answers is not in the document]
Comment (linsun) [reply content/en/docs/ambient/install/platform-prerequisites/index.md:54]: This was copied from https://istio.io/latest/docs/setup/platform-setup/k3d/.  I believe it is to use our Istio ingress which Trasefik serves similar function.

[the remark this answers is not in the document]
Comment (linsun) [reply content/en/docs/ambient/install/platform-prerequisites/index.md]: added a clarification for this.

**Comment by linsun** [comment]
removing DND as feedback has been addressed

**Pull request merged**